console/src/main/java/org/georchestra/console/ws/AreaController.java

```diff
@@ -0,0 +1,85 @@
+package org.georchestra.console.ws;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.util.StreamUtils;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+
+/**
+ * A simple controller to serve an area.json file from your datadir.
+ */
+@Controller
+public class AreaController {
+
+    @Value("${AreasUrl:area.geojson}")
+    private String areasUrl;
+
+    @Value("${georchestra.datadir:/etc/georchestra}")
+    private String datadir;
```

Comment:
If the datadir is not defined, then I'd use a default area.json nested in the webapp and serve it instead. But since we now consider that a georchestra datadir is mandatory, I wonder if it's not over-designing.

Reply:
> I wonder if it's not over-designing.

Indeed.
Such a file is pure personalization : a useful default cannot be provided